### DISCUSSION TITLE
Selected spot bigger radius

### DISCUSSION THREAD

**Discussion by FMorschel** [imaNNeo/fl_chart#1490] [Q&A] [answered]
Hi!

When the `handleBuiltInTouches` is set to `true`, when the tooltip is showing, the selected spots also get bigger by default. 

This is handled by `getTouchedSpotIndicator` from what I could tell. 

Is there any way for us to programmatically select some spots, by `showingTooltipIndicators` for example, and also make it so that the actual spots are drawn bigger on the screen?

**Comment by imaNNeo** [comment, marked as answer]
If you need to modify the default behavior, you must implement your approach yourself. And yes you can override the `touchCallback` and respond to the touch event as you want. (for example you can have some flags and do `setState()` to apply your own effect on the chart)

**Comment by FMorschel** [comment]
I'll dedicate some time to this when I find that my modifications are good enough for me, I'll post a minimal reproducible code here so that other people can use it
